main.py

```diff
@@ -0,0 +1,62 @@
+import tkinter as tk
+from tkinter import ttk
+from tkinter import filedialog
+import matplotlib.pyplot as plt
+import json
+
+# set color values
+light_blue = '#b8eef5'
+light_yellow = '#fcffd1'
+darker_yellow = '#fcffd1'
+
+
+# matplotlib logic
+class PlotView:
+    def __init__(self, file_name):
```

Comment:
You should add docstrings to all of the methods in this file. https://peps.python.org/pep-0008/#documentation-strings